modules/i18n/src/translations/en-US/portals/applications.ts

```diff
@@ -1181,6 +1181,36 @@ export const applications: ApplicationsNS = {
                 }
             }
         },
+
+        applicationsSettings: {
+            fields: {
+                dcrEndpoint: {
+                    label: "DCR Endpoint",
+                    placeholder: "https://dev.api.asgardeo.io/t/john/api/identity/oauth2/dcr/v1.1/register",
```

Comment:
Let's not add URLs in i18n files. We can directly use the URL as placeholder in code.

Also, please make sure to get the base URL through config, instead of hardcoding.

Reply:
Removed this placeholder with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3) since it is not needed. We are fetching the DCR endpoint url from discovery endpoint